app/src/main/java/org/hackillinois/android/view/IndoorMapsFragment.java

```diff
@@ -46,13 +50,13 @@ public void onTabReselected(TabLayout.Tab tab) {
     private void setImage(int index) {
         switch (index) {

```

Comment:
If you want to save a lot of lines of code, you could make a dictionary with entries like `DCL -> R.drawable.dcl_indoor` and query that with the index